'

```diff
@@ -0,0 +1,89 @@
+# CloudTrail
```

Comment:
Wait a second. What is this file? It's just called `'`?

Reply:
i removed this file it was a scratch from logging.tf